docs/release-notes/2.4/2.4.29.md

```diff
@@ -0,0 +1,27 @@
+# Percona XtraBackup 2.4.29 (2023-12-)
+
+!!! important
+
+    Percona XtraBackup 2.4.29 is the final release of the Percona XtraBackup 2.4 series.
+
+    We recommend that you either [upgrade to MySQL 8.0 or stay on 5.7; we'll support you](https://www.percona.com/navigating-mysql-5-7-end-of-life).
+
+[Percona XtraBackup for MySQL Databases](https://www.percona.com/software/mysql-database/percona-xtrabackup) enables MySQL backups without blocking user queries. Percona XtraBackup is ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, Percona XtraBackup drives down backup costs while providing unique features for MySQL backups.
```

Comment:
Remove the blurb except for the first line.

Reply:
fixed